2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Configure stats pid to 32184
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from /Users/hugoromero/.config/wandb/settings
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/settings
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'workspace/main.py', 'program': '/Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/main.py'}
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_init.py:_log_setup():506] Logging user logs to /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/run-20230927_100803-bmml2xxy/logs/debug.log
2023-09-27 10:08:03,157 INFO    MainThread:32184 [wandb_init.py:_log_setup():507] Logging internal logs to /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/run-20230927_100803-bmml2xxy/logs/debug-internal.log
2023-09-27 10:08:03,158 INFO    MainThread:32184 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x246367c40>
2023-09-27 10:08:03,158 INFO    MainThread:32184 [wandb_init.py:init():546] calling init triggers
2023-09-27 10:08:03,158 INFO    MainThread:32184 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'consequent_value': 'beef', 'fitness_function': 'composed_fitness_length_fitness', 'generation_size': 50, 'num_generations': 10, 'seed': 43, 'probability_of_zero': 0.6, 'mutation_probability': 0.001}
2023-09-27 10:08:03,158 INFO    MainThread:32184 [wandb_init.py:init():602] starting backend
2023-09-27 10:08:03,158 INFO    MainThread:32184 [wandb_init.py:init():606] setting up manager
2023-09-27 10:08:03,160 INFO    MainThread:32184 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-09-27 10:08:03,161 INFO    MainThread:32184 [wandb_init.py:init():613] backend started and connected
2023-09-27 10:08:03,163 INFO    MainThread:32184 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2023-09-27 10:08:03,163 INFO    MainThread:32184 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-09-27 10:08:03,163 INFO    MainThread:32184 [wandb_init.py:init():701] updated telemetry
2023-09-27 10:08:03,235 INFO    MainThread:32184 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-09-27 10:08:04,003 INFO    MainThread:32184 [wandb_run.py:_on_init():2133] communicating current version
2023-09-27 10:08:04,095 INFO    MainThread:32184 [wandb_run.py:_on_init():2142] got version response upgrade_message: "wandb version 0.15.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-09-27 10:08:04,095 INFO    MainThread:32184 [wandb_init.py:init():789] starting run threads in backend
2023-09-27 10:08:07,761 WARNING MainThread:32184 [wandb_init.py:init():1165] interrupted
Traceback (most recent call last):
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1144, in init
    run = wi.init()
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 801, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Users/hugoromero/anaconda3/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/hugoromero/anaconda3/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-09-27 10:08:13,563 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Configure stats pid to 32184
2023-09-27 10:08:13,563 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from /Users/hugoromero/.config/wandb/settings
2023-09-27 10:08:13,563 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/settings
2023-09-27 10:08:13,564 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-09-27 10:08:13,564 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-09-27 10:08:13,564 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'workspace/main.py', 'program': '/Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/main.py'}
2023-09-27 10:08:13,564 INFO    MainThread:32184 [wandb_init.py:_log_setup():506] Logging user logs to /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/run-20230927_100813-wzruiz7c/logs/debug.log
2023-09-27 10:08:13,564 INFO    MainThread:32184 [wandb_init.py:_log_setup():507] Logging internal logs to /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/run-20230927_100813-wzruiz7c/logs/debug-internal.log
2023-09-27 10:08:13,564 INFO    MainThread:32184 [wandb_init.py:init():546] calling init triggers
2023-09-27 10:08:13,565 INFO    MainThread:32184 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'consequent_value': 'beef', 'fitness_function': 'composed_fitness_length_fitness', 'generation_size': 50, 'num_generations': 5, 'seed': 43, 'probability_of_zero': 0.6, 'mutation_probability': 0.001}
2023-09-27 10:08:13,565 INFO    MainThread:32184 [wandb_init.py:init():602] starting backend
2023-09-27 10:08:13,565 INFO    MainThread:32184 [wandb_init.py:init():606] setting up manager
2023-09-27 10:08:13,568 INFO    MainThread:32184 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-09-27 10:08:13,570 INFO    MainThread:32184 [wandb_init.py:init():613] backend started and connected
2023-09-27 10:08:13,572 INFO    MainThread:32184 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2023-09-27 10:08:13,572 INFO    MainThread:32184 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-09-27 10:08:13,572 INFO    MainThread:32184 [wandb_init.py:init():701] updated telemetry
2023-09-27 10:08:13,655 INFO    MainThread:32184 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-09-27 10:08:14,109 INFO    MainThread:32184 [wandb_run.py:_on_init():2133] communicating current version
2023-09-27 10:08:14,215 INFO    MainThread:32184 [wandb_run.py:_on_init():2142] got version response upgrade_message: "wandb version 0.15.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-09-27 10:08:14,215 INFO    MainThread:32184 [wandb_init.py:init():789] starting run threads in backend
2023-09-27 10:08:44,341 INFO    MainThread:32184 [wandb_run.py:_console_start():2114] atexit reg
2023-09-27 10:08:44,341 INFO    MainThread:32184 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-09-27 10:08:44,341 INFO    MainThread:32184 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-09-27 10:08:44,341 INFO    MainThread:32184 [wandb_run.py:_redirect():2059] Redirects installed.
2023-09-27 10:08:44,342 INFO    MainThread:32184 [wandb_init.py:init():831] run started, returning control to user process
2023-09-27 10:09:02,526 INFO    MainThread:32184 [wandb_run.py:_finish():1854] finishing run 100386322/cooked_groceries.csv/wzruiz7c
2023-09-27 10:09:02,526 INFO    MainThread:32184 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-09-27 10:09:02,527 INFO    MainThread:32184 [wandb_run.py:_restore():2066] restore
2023-09-27 10:09:02,527 INFO    MainThread:32184 [wandb_run.py:_restore():2072] restore done
2023-09-27 10:09:04,871 WARNING MsgRouterThr:32184 [router.py:_handle_msg_rcv():114] No listener found for msg with uuid f2d435201be341978ae7f3d3d383e743 (control {
  req_resp: true
  local: true
  relay_id: "b0f48fa61eed4017be9d61b34d474134"
}
uuid: "f2d435201be341978ae7f3d3d383e743"
response {
  log_artifact_response {
    artifact_id: "QXJ0aWZhY3Q6NTg1Njg1ODQ4"
  }
}
)
2023-09-27 10:09:04,871 WARNING MsgRouterThr:32184 [router.py:message_loop():77] message_loop has been closed
2023-09-27 10:09:07,055 WARNING MsgRouterThr:32184 [router.py:_handle_msg_rcv():114] No listener found for msg with uuid 00fdca39e8be441f8ac543a5f5b4819d (control {
  req_resp: true
  local: true
  relay_id: "b0f48fa61eed4017be9d61b34d474134"
}
uuid: "00fdca39e8be441f8ac543a5f5b4819d"
response {
  log_artifact_response {
    artifact_id: "QXJ0aWZhY3Q6NTg1Njg1ODcx"
  }
}
)
2023-09-27 10:10:49,039 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Configure stats pid to 32184
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from /Users/hugoromero/.config/wandb/settings
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/settings
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'workspace/main.py', 'program': '/Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/main.py'}
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_init.py:_log_setup():506] Logging user logs to /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/run-20230927_101049-2t1wh63m/logs/debug.log
2023-09-27 10:10:49,040 INFO    MainThread:32184 [wandb_init.py:_log_setup():507] Logging internal logs to /Users/hugoromero/Documents/GitHub/catedra-uc3m-sementacion/workspace/wandb/run-20230927_101049-2t1wh63m/logs/debug-internal.log
2023-09-27 10:10:49,041 INFO    MainThread:32184 [wandb_init.py:init():546] calling init triggers
2023-09-27 10:10:49,041 INFO    MainThread:32184 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'consequent_value': 'beef', 'fitness_function': 'composed_fitness_length_fitness', 'generation_size': 50, 'num_generations': 10, 'seed': 43, 'probability_of_zero': 0.6, 'mutation_probability': 0.001}
2023-09-27 10:10:49,041 INFO    MainThread:32184 [wandb_init.py:init():602] starting backend
2023-09-27 10:10:49,041 INFO    MainThread:32184 [wandb_init.py:init():606] setting up manager
2023-09-27 10:10:49,043 INFO    MainThread:32184 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-09-27 10:10:49,044 INFO    MainThread:32184 [wandb_init.py:init():613] backend started and connected
2023-09-27 10:10:49,046 INFO    MainThread:32184 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2023-09-27 10:10:49,046 INFO    MainThread:32184 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-09-27 10:10:49,046 INFO    MainThread:32184 [wandb_init.py:init():701] updated telemetry
2023-09-27 10:10:49,119 INFO    MainThread:32184 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-09-27 10:11:01,402 WARNING MainThread:32184 [wandb_init.py:init():1165] interrupted
Traceback (most recent call last):
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1144, in init
    run = wi.init()
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 744, in init
    result = run_init_handle.wait(
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/hugoromero/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Users/hugoromero/anaconda3/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/hugoromero/anaconda3/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
